### PR TITLE
refresh feed (homepage) (bug 1103208)

### PR DESCRIPTION
**Pull request by ngokevin**
- removed mini_tile to use global app tile
- split out some of the CSS, unspecified some rules, removed unused css in feed.styl

**Note:** since some of the background image rules live in a folder, you won't be able to see the brand icons unless you run with an include.css. ```make build && MKT_COMPILED=1 make serve```

screen shot 2014-12-30 at 6 50 46 am
screen shot 2014-12-30 at 6 50 54 am
screen shot 2014-12-30 at 6 50 59 am

### CODE REVIEW

**Comment by ngokevin** [comment]
@spasovski

File: src/media/css/feed.styl [hunk 1]
```diff
@@ -1,859 +0,0 @@
-/* This is a shared file between Fireplace and Transonic. */
```
Comment (spasovski) [inline src/media/css/feed.styl]: Glad to see this gone even though some of it was moved elsewhere.

File: src/templates/_macros/feed_item.html [hunk 2]
```diff
@@ -5,30 +5,6 @@
 {% from '_macros/stars.html' import stars %}
 
 
-{% macro mini_tile(app, price_only, src) %}
```
Comment (spasovski) [inline src/templates/_macros/feed_item.html]: Yes!

File: src/templates/_macros/feed_item.html [hunk 3]
```diff
@@ -78,19 +54,16 @@ <h1 class="name">{{ feed_app.app.name|translate(feed_app) }}</h1>
...
+    <a class="brand-header"
+       data-brand-color="{{ color }}" data-brand-type="{{ brand.type }}"
+       href="{{ url('feed/feed_brand', [brand.slug])|urlparams(src=src) }}">
+      <h3>{{ feed.get_brand_type(brand.type, brand.apps.length) }}</h3>
+    </a>
+    <ul>
       {% for app in brand.apps %}
         {% if landing or (loop.index0 < max_apps) %}
-          {{ mini_tile(app, price_only=not landing and brand.layout == 'grid', src=src) }}
+          {{ app_tile(app, price_only=not landing and brand.layout == 'grid', src=src) }}
```
Comment (spasovski) [inline src/templates/_macros/feed_item.html]: `app_tile` doesn't have `price_only` but looking at the mocks maybe it should

File: src/templates/_macros/feed_item.html [hunk 4]
```diff
@@ -230,10 +200,10 @@ <h1 class="name">{{ shelf.name|translate(shelf)|safe }}</h1>
             {% for group in feed.group_apps(coll.apps) %}
               <div class="detail-item">
                 <section class="feed-layout-grid multi-app-tile mkt-tile grouped c">
-                  <h1 class="feed-tile-header grouped">{{ group.name }}</h1>
-                  <ul class="app-list">
+                  <h1 class="feed-item-header grouped">{{ group.name }}</h1>
+                  <ul>
                     {% for app in group.apps %}
-                      {{ mini_tile(app, price_only=not landing) }}
+                      {{ app_tile(app, price_only=not landing) }}
```
Comment (spasovski) [inline src/templates/_macros/feed_item.html]: same as above

**Comment by spasovski** [comment]
Font-size on 'free' seems off:



In the mocks:

**Comment by spasovski** [comment]
r+wc

File: src/templates/_macros/feed_item.html (hunk 3 above)
Comment (ngokevin) [inline src/templates/_macros/feed_item.html]: CSS (can) handle(s) it

**Comment by ngokevin** [comment]
thanks!

**Pull request merged**